src/statusMessages.ts

```diff
@@ -16,6 +16,18 @@ export namespace StatusMessages {
   export const NO_SUPPORTED_MANIFEST = `No supported manifest's file found to be analyzed.`;
   export const PYPI_INTERPRETOR_PATH =
     'Provide path for python interpretor `Code/File -> Preferences -> Settings -> Workspace Settings`.For details check READMEs';
-  export const PYPI_INTERPRETOR_CMD = `-c 'exec("""\nimport pkg_resources as pr;import json,sys;gd=pr.get_distribution;res=list();\nfor i in open(sys.argv[1]):\n    try:\n        rs={};I=gd(i);rs["package"]=I.key;rs["version"]=I.version;rs["deps"]=set();\n        for j in pr.require(i):\n            for k in j.requires():\n                K=gd(k);rs["deps"].add((K.key, K.version))\n        rs["deps"]=[{"package":p,"version":v}for p,v in rs["deps"]];res.append(rs)\n    except: pass\na=sys.argv[2:3]\nop=open(a[0],"w")if a else sys.stdout\njson.dump(res,op)\n""")'`;
+  export const PYPI_INTERPRETOR_CMD = `
```

Comment:
There is no change in the logic of the script, I just beautified it for readability.